Fix up formatting and typos in 'Helper Functions' (#107)
Ada-Developers-Academy/ada-precourse-v2: intro-to-python/helper-functions.md

Reviewer: Line 530 is
```
- `convert_km_to_mi` to convert the `speed` to a mi/hr
```

But the unit abbreviation should probably be expanded as it was in the intro
```
- `convert_km_to_mi` to convert the `speed` to miles per hour
```

1.  Start by writing a function called `get_number_from_user`
    * Then pull all of the pieces of code in `guess_the_number` that have to do with getting user input into `get_number_from_user`.  
    * Include any conditional statements that validate user input as a number.  
    * This function should ask the user for a number and then give an error message if the user inputs anything other than a number.  

>     * Last, it should return the valid user input, or None if there was no valid input.  

Reviewer: ```suggestion
    * Last, it should return the valid user input, or `None` if there was no valid input.  
```

>     * In `guess_the_number`, call this function and store the result in user_input.

Reviewer: ```suggestion
    * In `guess_the_number`, call `get_number_from_user` and store the result in `user_input`.
```

* title: Speeding

##### !question
You are driving a car from another country in the United States. The speed limit is posted in miles per hour, but your car speedometer shows your speed in kilometers per hour. 

Write a function `am_i_speeding` that takes in a `speed` in units of kilometers per hour and a `speed_limit` in units of miles per hour. The function `am_i_speeding` should return 
- `True` if you are speeding
- `False` if you are are not speeding
- `None` if `speed` or `speed_limit` is not a float or an int.

The function `am_i_speeding` should use the following provided helper functions: 

> - `convert_km_to_mi` to convert the `speed` to a miles per hour

Reviewer: ```suggestion
- `convert_km_to_mi` to convert the `speed` to miles per hour
```

* title: Speeding

##### !question

> You are driving a car from another country in the United States. The speed limit limit is posted in miles per hour, but your car speedometer shows your speed in kilometers per hour. 

Reviewer: ```suggestion
You are driving a car from another country in the United States. The speed limit is posted in miles per hour, but your car speedometer shows your speed in kilometers per hour. 
```
